### PR TITLE
Added some missing discovery kernel options

### PR DESCRIPTION
**Pull request by lzap**
These are the misisng ones from https://theforeman.org/plugins/foreman_discovery/15.0/index.html#3.1.5Discoveryimagekerneloptions

I skipped those which are not relevant anymore or I don't want them to be documented.

### CODE REVIEW

File: guides/doc-Provisioning_Guide/topics/con_unattended-use-and-customization.adoc [hunk 1]
```diff
@@ -59,6 +59,26 @@ fdi.ipv6.method::
...
+  Filenames with extensions to be downloaded and started during boot.
+For more information, see see xref:Extending_the_Discovery_Image[].
+
+fdi.zipserver::
+  TFTP server to use to download extensions from.
+For more information, see see xref:Extending_the_Discovery_Image[].
+
+fdi.countdown::
+  Number of seconds to wait until text-user interface is refreshed after the initial discovery attempt.
+This value defaults to 45 second.
```
Comment (melcorr) [inline guides/doc-Provisioning_Guide/topics/con_unattended-use-and-customization.adoc:72]:
```suggestion
This value defaults to 45 seconds.
```

File: guides/doc-Provisioning_Guide/topics/con_unattended-use-and-customization.adoc [hunk 2]
```diff
@@ -59,6 +59,26 @@ fdi.ipv6.method::
...
+fdi.zips::
+  Filenames with extensions to be downloaded and started during boot.
+For more information, see see xref:Extending_the_Discovery_Image[].
+
+fdi.zipserver::
+  TFTP server to use to download extensions from.
+For more information, see see xref:Extending_the_Discovery_Image[].
+
+fdi.countdown::
+  Number of seconds to wait until text-user interface is refreshed after the initial discovery attempt.
```
Comment (melcorr) [inline guides/doc-Provisioning_Guide/topics/con_unattended-use-and-customization.adoc:71]:
```suggestion
  Number of seconds to wait until the text-user interface is refreshed after the initial discovery attempt.
```

File: guides/doc-Provisioning_Guide/topics/con_unattended-use-and-customization.adoc [hunk 3]
```diff
@@ -59,6 +59,26 @@ fdi.ipv6.method::
...
+This value defaults to 45 second.
+Increase this value when status page reports N/A.
```
Comment (melcorr) [inline guides/doc-Provisioning_Guide/topics/con_unattended-use-and-customization.adoc:73]:
```suggestion
Increase this value when status page reports N/A.
```

Is N/A the exact thing that is reported? Or is there some error, or just a hanging black void that stares back at the user as the user stares into it?

File: guides/doc-Provisioning_Guide/topics/con_unattended-use-and-customization.adoc [hunk 4]
```diff
@@ -59,6 +59,26 @@ fdi.ipv6.method::
...
+
+fdi.dhcp_timeout::
+  NetworkManager DHCP timeout.
+This value default to 300 seconds.
```
Comment (melcorr) [inline guides/doc-Provisioning_Guide/topics/con_unattended-use-and-customization.adoc:77]:
```suggestion
The default value is 300 seconds.
```

**Review by melcorr — CHANGES_REQUESTED** [review]
Few small Czechlishes in an otherwise flawless PR

File: guides/doc-Provisioning_Guide/topics/con_unattended-use-and-customization.adoc (hunk 1 above)
Comment (melcorr) [reply guides/doc-Provisioning_Guide/topics/con_unattended-use-and-customization.adoc:72]: OR
```suggestion
The default value is 45 seconds.
```

File: guides/doc-Provisioning_Guide/topics/con_unattended-use-and-customization.adoc (hunk 3 above)
Comment (lzap) [reply guides/doc-Provisioning_Guide/topics/con_unattended-use-and-customization.adoc:73]: We get this reported quite often, TUI reads status of the background job and when it's not yet done (slow network) it prints IP Address: N/A.

**Comment by lzap** [comment]
How about this.

File: guides/doc-Provisioning_Guide/topics/con_unattended-use-and-customization.adoc (hunk 3 above)
Comment (melcorr) [reply guides/doc-Provisioning_Guide/topics/con_unattended-use-and-customization.adoc:73]: Cool, fine by me if that is what the user will see.

**Comment by lzap** [comment]
Sorry, how about now.

File: guides/doc-Provisioning_Guide/topics/con_unattended-use-and-customization.adoc [hunk 5]
```diff
@@ -68,13 +68,13 @@ fdi.zipserver::
 For more information, see see xref:Extending_the_Discovery_Image[].
 
 fdi.countdown::
-  Number of seconds to wait until text-user interface is refreshed after the initial discovery attempt.
-This value defaults to 45 second.
-Increase this value when status page reports N/A.
+  Number of seconds to wait until the text-user interface is refreshed after the initial discovery attempt.
+This value defaults to 45 seconds.
+Increase this value when status page reports IP address as N/A.
```
Comment (melcorr) [inline guides/doc-Provisioning_Guide/topics/con_unattended-use-and-customization.adoc:73]:
```suggestion
Increase this value if the status page reports the IP address as `N/A`.
```
Let's stay positive.

**Review by melcorr — COMMENTED** [review]
Can you implement my other suggestions and then we are good to go.

**Comment by lzap** [comment]
Done.

**Review by melcorr — APPROVED** [review]
Looks great thank you!

**Pull request merged**